README.md

```diff
@@ -21,6 +21,9 @@ In `mongodb.env`, add:
 - `MONGO_INITDB_ROOT_USERNAME=...`
 - `MONGO_INITDB_ROOT_PASSWORD=...`
 
+In `frontend.env`, add:
+- `REACT_APP_BASE_URL=http://localhost:8000/`
```

Comment:
I'm glad that you have included this as a required environment variable; however, could you add a small sentence/description as to what this should represent? Based on this, it seems to be the URL that the react app is being hosted on, rather than necessarily the API endpoint (for example if the API is running on a different domain).

Reply:
So include something like 
```suggestion
- `REACT_APP_BASE_URL=http://localhost:8000/` (this should be ...)
```